apm-agent-core/src/test/java/co/elastic/apm/agent/report/IntakeV2ReportingEventHandlerTest.java

```diff
@@ -147,7 +147,7 @@ void testUrls() throws MalformedURLException {
     }
 
     @Test
-    void testReport() {
+    void testReport() throws Exception {
```

Comment:
These tests use `reportTransaction`, which if you go deep enough you'll see using `IntakeV2ReportingEventHandler#dispatchEvent()` that now started using `IntakeV2ReportingEventHandler#startRequest()`, which `throws Exception`. I could catch it somewhere along the way, but I prefer it would be handled by the disruptor, like any other event handling error would.
You can see that `com.lmax.disruptor.EventHandler#onEvent`, which invokes `dispatchEvent()`, also `throws Exception`, so it is expected by the disruptor API.